frontend/common/src/constants/localizedConstants.tsx

```diff
@@ -25,25 +25,34 @@ import {
 } from "../api/generated";
 import { getOrThrowError } from "../helpers/util";
 
-export const womanLocalized = {
-  defaultMessage: "I identify as a woman",
-  description: "Message for woman",
-};
-
-export const indigenousLocalized = {
-  defaultMessage: "I am indigenous",
-  description: "Message for indigenous",
-};
-
-export const minorityLocalized = {
-  defaultMessage: "I identify as a member of a visible minority group",
-  description: " Message for minority group",
-};
+export const employmentEquityStatements = defineMessages({
+  woman: {
+    defaultMessage: '"I identify as a woman"',
+    description: "Statement for when someone indicates they are a woman",
+  },
+  indigenous: {
+    defaultMessage: '"I am Indigenous"',
+    description: "Statement for when someone indicates they are indigenous",
+  },
+  minority: {
+    defaultMessage: '"I identify as a member of a visible minority"',
+    description:
+      "Statement for when someone indicates they are a visible minority",
+  },
+  disability: {
+    defaultMessage: '"I identify as a person with a disability"',
+    description: "Statement for when someone indicates they have a disability",
+  },
+});
 
-export const disabilityLocalized = {
-  defaultMessage: "I identify as a person with a disability",
-  description: "Message for person with a disability",
-};
+export const getEmploymentEquityStatement = (
+  equityStatement: string,
```

Comment:
Could we further constrain this type beyond string to make sure that a missing key is a compile-time error instead of runtime?  Something like `keyof employmentEquityStatements` or make a union type or enum or a set of constants?